packages/dmn-editor/tests/e2e/drgElements/typeAttribution.spec.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { test, expect } from "../__fixtures__/base";
+import { DEFAULT_DRD_NAME } from "../__fixtures__/diagram";
+import { DataType } from "../__fixtures__/jsonModel";
+import { DefaultNodeName, NodeType } from "../__fixtures__/nodes";
+
+test.beforeEach(async ({ editor }) => {
+  await editor.open();
+});
+
+test.describe("Type attribution", () => {
+  test.describe("Input Data", () => {
+    test.only("should change Input Data data type", async ({ page, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.INPUT_DATA, targetPosition: { x: 100, y: 100 } });
+
+      await nodes.hover({ name: DefaultNodeName.INPUT_DATA });
+      await nodes.get({ name: DefaultNodeName.INPUT_DATA }).getByLabel(DataType.Undefined).click();
+      await page.pause();
+      await nodes
+        .get({ name: DefaultNodeName.INPUT_DATA })
+        .getByRole("option")
+        .getByText(DataType.Number, { exact: true })
+        .click();
+
+      await page.pause();
+
+      await nodes.hover({ name: DefaultNodeName.INPUT_DATA });
+      await page.pause();
+    });
```

Comment:
I am struggling to select the dropdown option, as we have dropdown implemented using `ul` and `li` instead of `select` and `option`, while playwright has a native support for the later one.

On the line `36` we stopped the debugger after click on the `<Undefined>`, so far so good, the dropdown was shown.
![Screenshot 2024-03-14 121918](https://github.com/ljmotta/kie-tools/assets/8044780/338e2487-34d0-4cdd-a9c4-b58b5b8fe6c3)

Then if we resume the test execution, and we check in the debug console where the next click was executed, it is in some inappropriate location, see the red circle:
![Screenshot 2024-03-14 122330](https://github.com/ljmotta/kie-tools/assets/8044780/a3cac2aa-68b3-4484-8136-2fc1afe3d169)